contracts/facets/ViewStateFacet.sol

```diff
@@ -249,7 +249,36 @@ contract ViewStateFacet {
         view
         returns (LibTypes.Sarcophagus memory)
     {
-        return s.sarcophagi[sarcoId];
+        LibTypes.Sarcophagus memory sarco = s.sarcophagi[sarcoId];
+        if (sarco.state != LibTypes.SarcophagusState.Active) {
+            return sarco;
+        }
```

Comment:
@katien -- there is an edge case here that isn't handled --

A Sarcophagus could be both `Cleaned` and `Resurrected`, or `Cleaned` and `Failed`. Not sure the best way to handle this -- it's more important the front end shows the `Resurrected`/`Failed` states, but would need to gate the `Clean` function from showing if the sarco has also been cleaned.